pkg/reconciler/autoscaling/kpa/resources/decider_test.go

```diff
@@ -130,6 +130,18 @@ func TestMakeDecider(t *testing.T) {
 		name: "with metric annotation",
 		pa:   pa(WithMetricAnnotation("rps")),
 		want: decider(withTarget(100.0), withPanicThreshold(2.0), withTotal(100), withMetric("rps"), withMetricAnnotation("rps")),
+	}, {
+		name: "with initial scale annotation",
+		pa:   pa(WithPAInitialScale("5")),
+		want: decider(withTarget(100.0), withPanicThreshold(2.0), withTotal(100), withInitialScale(5), withInitialScaleAnnotation("5")),
+	}, {
+		name: "use cluster initial scale",
+		cfgOpt: func(c autoscalerconfig.Config) *autoscalerconfig.Config {
+			c.InitialScale = 3
+			return &c
+		},
+		pa:   pa(),
+		want: decider(withTarget(100.0), withPanicThreshold(2.0), withTotal(100), withInitialScale(3)),
```

Comment:
should probably have a test for what happens if the annotation is invalid/zero since we have logic for that (i.e. the Atoi and the `if revInitScaleInt > 1` branches)